sdk/tables/tests.yml

```diff
@@ -5,13 +5,10 @@ stages:
     parameters:
       BuildTargetingString: azure-data-tables
       ServiceDirectory: tables
+      SupportedClouds: 'Public,UsGov,China'
       MatrixReplace:
         - TestSamples=.*/true
       EnvVars:
-        AZURE_TENANT_ID: $(aad-azure-sdk-test-tenant-id)
-        AZURE_SUBSCRIPTION_ID: $(azure-subscription-id)
-        AZURE_CLIENT_ID: $(aad-azure-sdk-test-client-id)
-        AZURE_CLIENT_SECRET: $(aad-azure-sdk-test-client-secret)
```

Comment:
You may still need:

```
        AZURE_TENANT_ID: $(TABLES_TENANT_ID)
        AZURE_SUBSCRIPTION_ID: $(TABLES_SUBSCRIPTION_ID)
        AZURE_CLIENT_ID: $(TABLES_CLIENT_ID)
        AZURE_CLIENT_SECRET: $(TABLES_CLIENT_SECRET)
```

Reply:
No, they will set when run scripts in test-resources.json. I have similar updates for [container registry](https://github.com/Azure/azure-sdk-for-python/pull/27203), tests are passing in all three clouds.

For issues here, I think we need to update `test-resources.json` file. 
- in UsGov: it can set the endpoint suffix dedicated for this cloud - `core.usgovcloudapi.net`([link](https://dev.azure.com/azure-sdk/internal/_build/results?buildId=1959616&view=logs&j=df77640d-ea76-562f-46b4-dd707f7236ef&t=ff0f47cf-e93c-557e-fd22-f0f182834652&l=97)),  but the suffix in failing tests are [secondary.table.core.windows.net](https://dev.azure.com/azure-sdk/internal/_build/results?buildId=1959616&view=logs&j=df77640d-ea76-562f-46b4-dd707f7236ef&t=46c2be30-b73e-5803-bded-79cd474ebee5&l=292151) which caused `ServiceRequestError`. Don't know how to update the secondary suffix.
- in China: failed at `Deploy test resources` while creating database account([link](https://dev.azure.com/azure-sdk/internal/_build/results?buildId=1959616&view=logs&j=40afa136-fb5d-5bfc-aac7-a72f6eb88f2c&t=80f02b9a-97dd-55da-e89b-ed000ea2d96b&l=61)), it lacks region access permission in our test subscription. @benbp Could you help on this?

Reply:
For china, I think we are hitting quota issues in china east 2. Can you try changing the region? You can override the cloud config settings like:

```
parameters:
  CloudConfig:
      Public:
        SubscriptionConfiguration: $(sub-config-azure-cloud-test-resources)
      UsGov:
        SubscriptionConfiguration: $(sub-config-gov-test-resources)
      China:
        SubscriptionConfiguration: $(sub-config-cn-test-resources)
        Location: chinanorth3
```

Reply:
Re: usgov suffix, perhaps the tests are constructing the secondary suffix without respecting the env variables that set the suffix?